stonesoup/updater/kalman.py

```diff
@@ -109,7 +109,7 @@ def _measurement_matrix(self, predicted_state=None, measurement_model=None,
             The measurement matrix, :math:`H_k`
 
         """
-        return self.measurement_model.matrix(**kwargs)
+        return measurement_model.matrix(**kwargs)
```

Comment:
The reason it's keyword is to cater for those situations where you want to use a measurement model contained within the Updater, rather than having one attached to each detection.

Reply:
See https://github.com/dstl/Stone-Soup/pull/148#issuecomment-557099891